src/css/text-styles.scss

```diff
@@ -106,10 +107,15 @@ $text-colors: (
 
 @mixin text__heading__two__separated {
   font-size: map-get($map: $text-sizes, $key: headingTwoSeparated);
-  line-height: map-get($map: $text-line-heights, $key: headingTwoSeparated);
+  line-height: map-get($map: $text-line-heights, $key: aside);
```

Comment:
typo error here

Reply:
Done

Reply:
The key es `aside`, is this correct?

Reply:
I don't think I use it anywhere. For me, I used it in a component that I had to delete, but, in this branch with main base, the text type does not appear.
![image](https://user-images.githubusercontent.com/107500715/221902062-2be6776d-7a7c-4795-ba6b-c0bfc8e3c842.png)